includes/config.php

```diff
@@ -21,7 +21,12 @@
 <span class="foost" data-process-id="{{_id}}"></span>
 <script type="text/javascript">
 
-	var pId = "{{_id}}";
+    var pId = "{{_id}}";
+
+	jQuery(".foost").siblings("#cf-convertkit-sequence-id").attr("id", "cf-convertkit-sequence-id-" + pId );
```

Comment:
@New0 Please change `foost` to something meaningful. This is a legacy mistake of mine that we can correct